erpnext/selling/doctype/customer/customer.py

```diff
@@ -403,11 +403,13 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
 				primary_action={
 					'label': 'Send Email',
 					'server_action': 'erpnext.selling.doctype.customer.customer.send_emails',
+					'hide_on_success': 1,
 					'args': {
 						'customer': customer,
 						'customer_outstanding': customer_outstanding,
 						'credit_limit': credit_limit,
-						'credit_controller_users_list': credit_controller_users
+						'credit_controller_users_list': credit_controller_users_list,
```

Comment:
```suggestion
						'credit_controller_users_list': credit_controller_users,
```
`credit_controller_users_list` is not defined